### ISSUE TITLE
ExecuteMany constructor error when converting to TypeScript

### ISSUE THREAD

**Issue by jaredbaszler** [oracle/node-oracledb#1294]
1. What versions are you using?

DB Version: Oracle: 10.2

Also run Node.js and show the output of:

Welcome to Node.js v14.10.0.
Type ".help" for more information.
> process.platform
'win32'
> process.version
'v14.10.0'
> process.arch
'x64'

oracleDB: v5.0.0
instant client version: 12.1.0.2.0

2. Is it an error or a hang or a crash?

Compile time error: 
![image](https://user-images.githubusercontent.com/9924084/92812601-2e3a6400-f385-11ea-92fa-4588da97cc40.png)

3. What error(s) or behavior you are seeing?

Before I converted to Typescript (`npm install @types/oracledb`), the code block was working just fine.  I was also successfully running various select, update and insert commands.  Including this `.executeMany()` call.  

Cut and paste text showing the command you ran.  No screenshots.

NONE

**Comment by jaredbaszler** [comment]
FYI - I cleaned up the rest of our project tonight and confirmed all of our other selects, updates and inserts are working except for the one above.

**Comment by jaredbaszler** [comment]
My next attempt got me past the compile time error but then a run time error of `Error: NJS-055: binding by position and name cannot be mixed` occurred.  That came from this:  
![image](https://user-images.githubusercontent.com/9924084/92950807-eff38200-f422-11ea-9000-b53900315065.png)

To get around that I did this and then it worked:
![image](https://user-images.githubusercontent.com/9924084/92950897-1f09f380-f423-11ea-917c-70ede81bd846.png)

Not sure if there is a better way to do all of this as I'm someone new to typescript and Node.js.  But I figured at the very least I'd leave this here to help anyone else.
